translations/en-us.all.json

```diff
@@ -1893,7 +1893,9 @@
   },
   {
     "id": "faq_ads_explanation",
-    "translation": "[lengthy explanation as to why we have ads"
+    "translation": "This is not a decision we make lightly, and we have thoroughly considered the objections raised by all of you, but ultimately we feel this is the only sustainable solution to fund Pantsu and ensure its continued operation. \
```

Comment:
Use <br> instead. JSON format doesn't allow line-breaks.

Reply:
I'm still keeping the \ because having an extremely long line isn't really good for translation purposes